tests/model_selection/splitters/pubchem_split.py

```diff
@@ -0,0 +1,250 @@
+from typing import Union
+from unittest.mock import patch
+
+import pytest
+from rdkit import Chem
+from rdkit.Chem import Mol
+
+from skfp.model_selection.splitters.pubchem_split import (
+    get_cid_for_smiles,
+    get_earliest_publication_date,
+    pubchem_train_test_split,
+    pubchem_train_valid_test_split,
+)
+
+
+@pytest.fixture
+def proper_smiles() -> str:
+    return "Cc1cc(cc(c1)Oc2ccc(cc2Br)Cc3c4ccnnc4[nH]n3)C#N"
+
+
+@pytest.fixture
+def smiles_that_cannot_be_standarized() -> str:
+    return "c1ccccc12cCc4"
+
+
+@pytest.fixture
+def smiles_that_not_exists_in_pubchem() -> str:
+    return "C[Fe](C)(C)OC(C)=O"
+
+
+@pytest.fixture
+def proper_pubchem_cid() -> str:
+    return "24261"
```

Comment:
If you have fixtures that are used only in a single place, then don't use a fixture, but rather provide the value explicitly. This brings it closer to actual usage, increasing readability